### PR TITLE
Integrations to packs Phase 2

### PR DESCRIPTION
**Pull request by liorblob**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

list of integrations:
ZeroFox
WindowsDefenderAdvancedThreatProtection
Whois
WhatIsMyBrowser
VulnDB
VMRay
VirusTotal-Private_API
Vertica
Vectra_v2
UrlScan

### CODE REVIEW

**Comment by Itay4** [comment]
so which phase is this ?

File: Packs/UrlScan/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "urlscan.io",
+    "description": "Urlscan.io reputation",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (IkaDemisto) [inline Packs/UrlScan/pack_metadata.json:5]: You can remove it, will generate it in the build

File: Packs/UrlScan/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "urlscan.io"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (IkaDemisto) [inline Packs/UrlScan/pack_metadata.json:20]: Same here

**Comment by liorblob** [comment]
phase 2 I guess

File: Packs/UrlScan/pack_metadata.json (hunk 2 above)
Comment (yuvalbenshalom) [inline Packs/UrlScan/pack_metadata.json:20]: please remove

File: Packs/VMRay/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "VMRay",
+    "description": "Ransomware analysis sandboxing.",
+    "support": "xsoar",
+    "serverMinVersion": "4.0.0",
```
Comment (yuvalbenshalom) [inline Packs/VMRay/pack_metadata.json:5]: please remove

File: Packs/VMRay/pack_metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "VMRay"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/VMRay/pack_metadata.json:20]: please remove

File: Packs/Vectra_v2/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Vectra v2",
+    "description": "Automated attacker behavior analytics",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Vectra_v2/pack_metadata.json:5]: please remove

File: Packs/Vectra_v2/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        "Network Security"
+    ],
+    "tags": [
+        "Vectra v2"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Vectra_v2/pack_metadata.json:20]: please remove

File: Packs/Vertica/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Vertica",
+    "description": "Analytic database management software",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Vertica/pack_metadata.json:5]: please remove

File: Packs/Vertica/pack_metadata.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        "Database"
+    ],
+    "tags": [
+        "Vertica"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Vertica/pack_metadata.json:20]: please remove

File: Packs/VirusTotal-Private_API/pack_metadata.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "VirusTotal - Private API",
+    "description": "Analyze suspicious hashes, URLs, domains and IP addresses",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/VirusTotal-Private_API/pack_metadata.json:5]: please remove

File: Packs/VirusTotal-Private_API/pack_metadata.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "VirusTotal - Private API"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/VirusTotal-Private_API/pack_metadata.json:20]: please remove

File: Packs/VulnDB/pack_metadata.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "VulnDB",
+    "description": "Lists all of the security vulnerabilities for various products (OS,Applications) etc)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/VulnDB/pack_metadata.json:5]: please remove

File: Packs/VulnDB/pack_metadata.json [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+        "Vulnerability Management"
+    ],
+    "tags": [
+        "VulnDB"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/VulnDB/pack_metadata.json:20]: please remove

File: Packs/WhatIsMyBrowser/pack_metadata.json [hunk 13]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "WhatIsMyBrowser",
+    "description": "Parse user agents and determine if they are malicious as well as enrich information about the agent",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/WhatIsMyBrowser/pack_metadata.json:5]: please remove

File: Packs/WhatIsMyBrowser/pack_metadata.json [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "WhatIsMyBrowser"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/WhatIsMyBrowser/pack_metadata.json:20]: please remove

File: Packs/Whois/pack_metadata.json [hunk 15]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Whois",
+    "description": "Provides data enrichment for domains.",
+    "support": "xsoar",
+    "serverMinVersion": "4.1.0",
```
Comment (yuvalbenshalom) [inline Packs/Whois/pack_metadata.json:5]: please remove

File: Packs/Whois/pack_metadata.json [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Whois"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Whois/pack_metadata.json:20]: please remove

File: Packs/WindowsDefenderAdvancedThreatProtection/pack_metadata.json [hunk 17]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Windows Defender Advanced Threat Protection (Deprecated)",
+    "description": "Deprected - use Microsoft Defender Advanced Threat Protection instead. Windows Defender Advanced Threat Protection (ATP) is a unified platform for preventative protection, post-breach detection, automated investigation, and response.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/WindowsDefenderAdvancedThreatProtection/pack_metadata.json:5]: please remove

File: Packs/WindowsDefenderAdvancedThreatProtection/pack_metadata.json [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+        "Endpoint"
+    ],
+    "tags": [
+        "Windows Defender Advanced Threat Protection (Deprecated)"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/WindowsDefenderAdvancedThreatProtection/pack_metadata.json:20]: please remove

File: Packs/ZeroFox/pack_metadata.json [hunk 19]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "ZeroFox",
+    "description": "Cloud-based SaaS to detect risks found on social media and digital channels.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
+    "currentVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/ZeroFox/pack_metadata.json:6]: please remove

File: Packs/ZeroFox/pack_metadata.json [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "ZeroFox"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/ZeroFox/pack_metadata.json:20]: please remove

File: Packs/VMRay/pack_metadata.json (hunk 3 above)
Comment (yuvalbenshalom) [reply Packs/VMRay/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.0.0",
```
@liorblob I think you removed "currentVersion": "1.0.0".
apply this to correct.

**Comment by liorblob** [comment]
@yuvalbenshalom Fails only on metadata validation. We can merge.

**Pull request merged**